### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by winiciuscota** [donotnoot/samlvpn#5]
Is it still working? I'm getting a error similar to what I get with the normal oenvpn client

I'm getting this error trying to connect to the vpn 
![image](https://github.com/donotnoot/samlvpn/assets/6284656/4973d230-a016-4ccf-894c-4f738bc65486)

I have patched and built openvpn like the instructions
![image](https://github.com/donotnoot/samlvpn/assets/6284656/d0c39f5c-387f-4928-9f6b-f556f1fd823f)

**Comment by donotnoot** [comment]
Hello. I'm not actively maintaining this, because I'm very luckily not working somewhere where I need this awful VPN anymore. There's also the terrible official client (that probably violates GPL lol) that works (sometimes) available for Linux, and a few distros carry it in their repos.

In any case, it looks like the error is coming from AWS themselves?

Give it a try with the official client and see if that works.

**Comment by winiciuscota** [comment]
This vpn used to work great with openvpn client before my company moved to a saml auth policy for the vpn. I can connect using openvpn3 but after that I cant access the vpn only websites of the company nor ping the nameservers added in resolv.conf by openvpn3.
I thought your project could be the solution since it uses the openvpn client, which worked for me in the past. The error message I'm getting is the same you I'll get if you try to access any saml ovpn with the normal openvpn client.
My current solution is to have a different windows machine running openvpn connect with a wsl ssh client, I then connect to this ssh client with sshuttle when I need to access the vpn only resources. My company doesn't support linux

**Comment by donotnoot** [comment]
What distro are you using? Also I guess you're trying to connect to AWS VPN right?

**Comment by winiciuscota** [comment]
I'm using manjaro, don't think the vpn is from AWS

**Comment by donotnoot** [comment]
Right, this has only been tested and is intended for AWS VPN, so if you're using something else you're going to have to do some hacking. Patches welcome!

**Comment by winiciuscota** [comment]
I see, I thought that the project supported saml vpns in general. Do you think there can be much difference from one another? maybe increasing the length even more could work
Btw do you still have access to an aws vpn to confirm if it still works there?

**Comment by donotnoot** [comment]
>Do you think there can be much difference from one another? maybe increasing the length even more could work

Yeah, they could be completely different. You'll need to figure out how your VPN works to adapt it. Increasing the lengths might work, might not work, who knows! Only one way to find out. But like I said, try to figure out how the Windows client for your VPN works and try to mimic that, that's how this project came to life.

>Btw do you still have access to an aws vpn to confirm if it still works there?

I'm afraid not.
